0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -217,15 +229,24 @@ Application layer protocols deal with the exchange of payment details and associ
 
 The Simple Payment Setup Protocol (SPSP) is an application layer protocol for negotiating payment details. SPSP handles account and amount discovery, condition creation, quoting and setup. SPSP uses Webfinger ([RFC 7033](https://tools.ietf.org/html/rfc7033)) and an HTTP-based protocol for querying account and amount details, [ILQP](#interledger-quoting-protocol-ilqp) for quoting, and [ILP](#interledger-protocol-ilp) for payment execution.
 
-The protocol is described in [IL-RFC 9](../0009-simple-payment-setup-protocol/).
+The protocol is described in [IL-RFC 9](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md).
```

Comment:
Note that we have to update the script that renders these specs for interledger.org. Currently, this link will link you to the unrendered `.md` file.

(But it is better to fix the link for Github users' benefit and then just rewrite it back as part of the rendering process for the website.)